frontend/src/services/fetchData.ts

```diff
@@ -83,11 +89,33 @@ export const fetchApplicationAdmins = async (): Promise<FamAppAdminGetResponse[]
  * @param applicationAdminId id to delete fam_user_role_assignment record.
  */
 export const deleteAndRefreshApplicationAdmin = async (
-    applicationAdminId: number,
+    applicationAdminId: number
 ): Promise<FamAppAdminGetResponse[]> => {
     await AdminMgmtApiService.applicationAdminApi.deleteApplicationAdmin(
         applicationAdminId
     );
     // When deletion is successful, refresh (fetch) for frontend state.
     return fetchApplicationAdmins();
 };
+
+export const fetchDelegatedAdmins = async (
+    applicationId: number | undefined
+): Promise<FamAccessControlPrivilegeGetResponse[]> => {
+    const delegatedAdmins = (
```

Comment:
Hi Jota, let's add if application id is undefined, return []

Reply:
sure, I'll add this condition